src/levanter/data/shard_cache.py

```diff
@@ -1640,6 +1640,13 @@ def _get_chunk_unmapped(self, mapped_index: int, *, timeout: Optional[float] = N
                     current_timeout *= 2
                     current_timeout = min(current_timeout, 100)
                     continue
+                except asyncio.exceptions.InvalidStateError:
+                    self.logger.warning(f"Invalid state waiting for chunk {mapped_index} for {int(next_time - time_in)} seconds")
```

Comment:
weird. can you please log the exception (and send me a log?)

Reply:
Sure, but it'll take me some time as I now have all my TPU quota in use.